service/audit-service/src/main/java/uk/gov/gchq/palisade/audit/service/request/AuditRequestProcessingComplete.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 2019 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.gov.gchq.palisade.audit.service.request;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+
+import uk.gov.gchq.palisade.Context;
+import uk.gov.gchq.palisade.RequestId;
+import uk.gov.gchq.palisade.ToStringBuilder;
+import uk.gov.gchq.palisade.User;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * This is one of the objects that is passed to the {@link uk.gov.gchq.palisade.audit.service.AuditService}
+ * to be able to store an audit record. This class extends {@link AuditRequest} This class
+ * is used for the indication to the Audit logs that processing has been completed.
+ */
+public class AuditRequestProcessingComplete extends AuditRequest {
```

Comment:
I think this would be better named ProcessingCompleteAuditRequest. My instinct when reading AuditRequestProcesssingComplete is that it is the audit request that has completed, not that you are auditing the fact that the data access request has been processed successfully. Also I don't think we need this class until updating the data service to apply the appropriate logging.